internal/oci/uvm.go

```diff
@@ -116,6 +119,8 @@ const (
 	annotationBootFilesRootPath          = "io.microsoft.virtualmachine.lcow.bootfilesrootpath"
 	annotationStorageQoSBandwidthMaximum = "io.microsoft.virtualmachine.storageqos.bandwidthmaximum"
 	annotationStorageQoSIopsMaximum      = "io.microsoft.virtualmachine.storageqos.iopsmaximum"
+	annotationDisableKernelDirectBoot    = "io.microsoft.virtualmachine.disablekerneldirect"
```

Comment:
This just hit me what you were doing here. Lets just make this:

```go
// annotationKernelDirectBoot controls the use of the `KernelDirect` feature for LCOW which is automatically
// enabled on builds `>= 18286`. To disable this behavior set `false`.
annotationKernelDirectBoot = "io.microsoft.virtualmachine.kerneldirect"
```

And then lower you would set:
lopts.KernelDirect = parseAnnotationsBool(ctx, s.Annotations, annotationKernelDirectBoot, lopts.KernelDirect)
if !lopts.KernelDirect {
lopts.KernelFile = uvm.KernelFile
}

Reply:
Its less confusing than a `disable if` pattern.

Reply:
Much better, good idea :)

Reply:
In this case, I'm going to take out the TODO for removing the annotation since we might want to have control over direct boot/uefi boot for something else in the future :)

Reply:
Deal